MiBench/automotive/bitcount/src/bitcnts.c

```diff
@@ -52,42 +49,49 @@ int main(int argc, char *argv[])
     "Non-recursive bit count by bytes (AR)",
     "Shift and count bits"
   };
-  if (argc<2) {
-    fprintf(stderr,"Usage: bitcnts <iterations>\n");
+  if (argc < 2) {
+    fprintf(stderr, "Usage: bitcnts <iterations>\n");
     exit(-1);
-	}
-  iterations=atoi(argv[1]);
-  
+  }
+  iterations = atoi(argv[1]);
+  start_main=clock();
   puts("Bit counter algorithm benchmark\n");
-  
+start1 = clock();
   for (i = 0; i < FUNCS; i++) {
-    start = seconds_now();
-    
+    start = clock();
+
     for (j = n = 0, seed = rand(); j < iterations; j++, seed += 13)
-	    n += pBitCntFunc[i](seed);
-    
-    stop = seconds_now();
+      n += pBitCntFunc[i](seed);
+
+    stop = clock();
     ct = (stop - start);
     if (ct < cmin) {
-	    cmin = ct;
```

Comment:
again, you have changed the format. I explicitly asked you earlier not to do this.